spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/ConditionalRejectingErrorHandler.java

```diff
@@ -69,10 +76,33 @@ public ConditionalRejectingErrorHandler(FatalExceptionStrategy exceptionStrategy
 		this.exceptionStrategy = exceptionStrategy;
 	}
 
+	/**
+	 * Set to false to disable the, now, default behavior of logging and discarding
```

Comment:
It reads OK to me; what is the objection?

The `, now,` is intended to emphasize that this default behavior is new `@since 2.1`.

cc/ @buzzardo - Jay, can you help us out? Thanks.

Reply:
Ah! I see what you mean.

In our language I would place that `now` into the brackets - `to disable (now) the default behavior`.

What confuses me here is a comma after `the` 😄

Reply:
I am ok with  "the (now) default" if you want to change it on merge and fix the others. Or I will fix tomorrow when I am home.